K10plus PICA JSON.js

```diff
@@ -651,8 +610,8 @@ function generalParser(picaObj, item, hasSuperiorWork, superiorWorkPicaObj, isSu
 /** BEGIN TEST CASES **/
 var testCases = [
 	{
-		"type": "import",
-		"input": "{\"ppn\":[\"88511759X\"], \"catalogid\": \"stabikat\"}",
+		"type": "search",
+		"input": "{\"ppn\": \"88511759X\"}",
```

Comment:
Will this actually work? I'm not sure it should be a string, since detectSearch/doSearch expect an object.

Reply:
Scaffold doesn't support search translators at all, so these tests wouldn't be doing anything now.

Reply:
Yeah, true, although you can run them with `testTranslators.html` in the browser connector (and they run automatically on CI). I think it's worthwhile to keep them - we want to have some kind of indicator of whether the translator is still working in the future.

Reply:
Ah, OK, yeah, I was about to ask that. We definitely want them to work, then.